src/actions/requests.rs

```diff
@@ -605,6 +623,8 @@ impl ExecuteCommand {
     }
 }
 
+/// Get a list of commands that can be applied to the current document. For
+/// example, commands to automatically fix a "did you mean blah?" suggestion.
```

Comment:
This is not quite right, a CodeAction is any action that is performed on a specific document and range of text by the server